### PR TITLE
fix(bug #112): `tokens.get()` memorization

### PR DESCRIPTION
**Pull request by enten**
Bug https://github.com/oddbird/accoutrement/issues/112 describes what is fixed.

### CODE REVIEW

**Review by mirisuzanne — CHANGES_REQUESTED** [review]
Nice catch, this fix looks great! Thank you for the contribution. Can you update the changelog as well, to document this fix?

**Comment by enten** [comment]
> Can you update the changelog as well, to document this fix?
Okay, but do we need to bump to version 4.0.2 in package.json too?

File: test/tokens/_api.scss [hunk 1]
```diff
@@ -34,6 +34,29 @@
       );
     }
 
+    @include it('Memorize resolved value') {
```
[the remark this answers is not in the document]
Comment (enten) [reply test/tokens/_api.scss]: Done

**Review by mirisuzanne — APPROVED** [review]
Looks good! We can update the version when we release it. Thanks!

**Pull request merged**